build.sbt

```diff
@@ -2223,6 +2223,22 @@ runEngineDistribution := {
   )
 }
 
+lazy val runProjectManagerDistribution =
+  inputKey[Unit](
+    "Run or --debug the project manager distribution with arguments"
+  )
+runProjectManagerDistribution := {
+  buildEngineDistribution.value
+  buildProjectManagerDistribution.value
```

Comment:
If we wanted to [run the project manager in JVM mode](https://github.com/enso-org/enso/pull/7181/files#r1248480723) we'd need to replace this `buildProjectManagerDistribution.value` with a dependency on _"something"_ produced by #7153. Do you know what that'd be, @hubertp?